### ISSUE TITLE
Feature: Generate changelog

### ISSUE THREAD

**Issue by loris** [dylang/npm-check#58]
It would be pretty handy if `npm-check` after updating the package.json could show the user a complete changelog (using https://github.com/dylang/changelog) of all the deps which got upgraded. Even better: it could automatically (could be a CLI option) generate a commit message.
(here is a project doing that king of stuff for PHP/composer: https://github.com/pyrech/composer-changelogs)

**Comment by dylang** [comment]
Yes, this! I made changelog and this with that intention, it's just been hard to find time to do it.

**Comment by panrafal** [comment]
Found `npm-check` through `changelog` and I must admit it would be uber great to have it!
